lib/duckdb/converter.rb

```diff
@@ -171,5 +184,12 @@ def integer_to_hugeint(value)
         raise(ArgumentError, "The argument `#{value.inspect}` must be Integer.")
       end
     end
+
+    def decimal_to_hugeint(value)
```

Comment:
Should we accept the following behavior?

In duckdb,
```
D create table t (value decimal(19, 17));
D insert into t values (0.99999999999999999);
D select * from t;
┌─────────────────────┐
│        value        │
│   decimal(19,17)    │
├─────────────────────┤
│ 0.99999999999999999 │
└─────────────────────┘
```

But I think
```ruby
stmt.bind_decimal(1, 0.99999999999999999)
```
is same as
```ruby
stmt.bind_decimal(1, 1)
```
and we can't insert 0.99999999999999999 but 1.

Do I misunderstand?

Reply:
Sorry, current ruby-duckdb

```ruby
con.execute("INSERT INTO t VALUES (?)", 0.99999999999999999);
```

inserts 1 but not 0.99999999999999999.

So, I think this is restriction.

Reply:
I'm so sorry I missed this behavior.
Although we can pass the decimal value like the following, I will try to figure it out.

```
con.execute("INSERT INTO t VALUES (?)", BigDecimal(0.99999999999999999));
```